src/test/java/org/kainos/groupA/integration/BandCompIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+package org.kainos.groupA.integration;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.groupA.GroupAApplication;
+import org.kainos.groupA.GroupAConfiguration;
+import org.kainos.groupA.models.Competency;
+import org.kainos.groupA.models.JobRole;
+
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class BandCompIntegrationTest {
+    static final DropwizardAppExtension<GroupAConfiguration> APP = new DropwizardAppExtension<>(
+            GroupAApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    @Test
+    void getCompByBandID_shouldReturnStatusCode200() {
+        Response response = APP.client().target("http://localhost:8080/api/band-comp/2")
+                .request()
+                .get(Response.class);
+        Assertions.assertEquals(response.getStatus(), HttpStatus.OK_200);
+    }
+
+    @Test
+    void getCompByBandID_shouldReturnStatusCode404() {
+        Response response = APP.client().target("http://localhost:8080/api/band-comp/string")
+                .request()
+                .get(Response.class);
+        Assertions.assertEquals(response.getStatus(), HttpStatus.NOT_FOUND_404);
+    }
+
+    @Test
+    void getCompByBandID_shouldReturnListOfCompetenciesForBand() {
```

Comment:
you should merge this test with getCompByBandID_shouldReturnStatusCode200 and check the code and the response values as part of 1 test to save time

Reply:
test merge with getCompByBandID_shouldReturnStatusCode200